BlogApp/Screens/Login.tsx

```diff
@@ -0,0 +1,147 @@
+import React, {useState} from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+
+import {useDispatch} from 'react-redux';
+import {loginUser, registerUser} from '../redux/userSlice';
+import {AppDispatch} from '../store';
+
+const Login = () => {
+  const [isRegister, setIsRegister] = useState(false);
+  const [userEmail, setUserEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [rePassword, setRePassword] = useState('');
+
+  const dispatch = useDispatch<AppDispatch>();
+
+  const handleLogin = () => {
+    if (userEmail && password) {
+      dispatch(loginUser({userEmail, password}));
+    } else {
+      console.log('Error');
+    }
+  };
+  const handleRegister = async () => {
```

Comment:
would be better to create a custom hook named `useBlogs` and add all the handleRegister, handleDel, handleEdit etc functions you created there and simply import the fucntion in your tsx files